docs/howto/custom-management-commands.txt

```diff
@@ -256,6 +256,15 @@ All attributes can be set in your derived class and can be used in
     because attempting to set the locale needs access to settings. This
     condition will generate a :class:`CommandError`.
 
+.. attribute:: BaseCommand.style
+
+    An instance attribute that defines the style of the output, when writing to,
+    e.g., stdout.
+
+    Used for emphasizing messages by adding a color to them. To avoid text
+    higlighting, pass the :djadminopt:`--no-color` option when running your
+    command. To modify the color palette, see the :ref:`syntax-coloring`.
```

Comment:
We can also see the list of available style options in the doc which I think would be good to mention.